tdx-attestation-app/src/HttpClient.cpp

```diff
@@ -0,0 +1,147 @@
+//-------------------------------------------------------------------------------------------------
```

Comment:
Why do we have a separate HTTP client? Can't we use the existing one from the library?

Reply:
This is one is calling the attestation provider, and its temporary. Once full Guest Atetstaion works this will be moved to the guest attestation library.